ocemr/templates/med_queue.html

```diff
@@ -48,7 +48,6 @@
 <TD>{{v.get_estimated_visit_cost}} (<A HREF=# onClick='window.open("/visit/{{v.id}}/cost_estimate_detail/","Cost Estimate Detail","menubar=no,width=540,height=420,toolbar=no,scrollbars=yes");'>detail</A>)</TD>
 <TD>{{v.cost}}(<A HREF=# onClick='window.open("/visit/{{v.id}}/bill_amount/","Collect","menubar=no,width=540,height=420,toolbar=no,scrollbars=yes");'>edit</A>)</TD>
 <TD>
-{% ifequal v.get_med_status 2 %}
 	{% if v.has_collected %}
```

Comment:
fix indentation here?

Reply:
Rejected. This matches other indentation in the file.